native-image-tests/src/main/resources/META-INF/native-image/native-image-tests/reflect-config.json

```diff
@@ -0,0 +1,215 @@
+[
```

Comment:
Idea 1: Work out what is needed for junit framework and check that in.

Reply:
Is it more robust to compute this by running a test?

Reply:
I was sort of imagining a combined approach.  Each different aspect would be independently correct.

1. Junit 5 launchers we could get the right settings via a one off manual process until Junit 5 supports graal inately.
2. Our tests would be configured overall via an AutomaticFeature separating out detection from running.  So you could use the image to run any set of tests etc.
3. We have correct okhttp main code that works automatically with graal.